pkgs/development/python-modules/graph-tool/2.x.x.nix

```diff
@@ -1,11 +1,13 @@
 { stdenv, fetchurl, python, cairomm, sparsehash, pycairo, autoreconfHook,
 pkgconfig, boost, expat, scipy, numpy, cgal, gmp, mpfr, lndir,
-gobjectIntrospection, pygobject3, gtk3, matplotlib }:
+gobjectIntrospection, pygobject3, gtk3, matplotlib, ncurses }:
 
 stdenv.mkDerivation rec {
-  version = "2.16";
+  version = "2.26";
   name = "${python.libPrefix}-graph-tool-${version}";
 
+  pythonModule = python;
```

Comment:
it's better to use `toPythonModule (stdenv.mkDerivation {...})`

Reply:
Even better would be to try and use `buildPythonPackage` with `format = "other";`

Reply:
Thanks a lot for the quick feedback. I did try `buildPythonPackage` earlier, but I did not know about `format = "other"`. With the additional `format` attribute it works. I only retested on OSX/Py3k tough as the compilation takes ages and I don't think that change will have an effect on the other combination, or will it?